components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -99,18 +99,11 @@ IB_DESIGNABLE
 @property(nonatomic, strong, nullable) UIView *titleView;
 
 /**
- Display attributes for the titleView's title text.
-
- Setting this property will render an NSAttributedString with the assigned attributes across the
- entire text.
+ The font applied to the title of navigation bar.
+ Font size is enforced to 20.
+ Default is MDCTypography's titleFont.
```

Comment:
Please document what happens when this property is set to nil.

Reply:
it's null_resetable to default, MDCTypography titleFont is nonnull.

Reply:
There are two things we need to document here:

1. What the default, initial value is.
2. What happens when we set the property to nil.